Sources/Qonversion/Qonversion.h

```diff
@@ -78,7 +78,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  Return Qonversion Products in assotiation with Store Kit Products
- 
+ If you get an empty SKProducts be sure your in-app purchases are correctly setted up in AppStore Connect and .storeKit file is available.
+
+ @see [Installing the iOS SDK](https://documentation.qonversion.io/docs/ios-sdk-setup)
```

Comment:
```suggestion
 @see [Installing the iOS SDK](https://qonversion.io/docs/apple)
```

That's a shortcut for the destination.